### PR TITLE
Add basic Bazel version updating

### LINKED ISSUE #65: Update bazel version
Bazel Steward should be able to detect newer version of bazel and suggest bumping it.

### PR DESCRIPTION
**Pull request by kamildoleglo**
Fixes #65

### CODE REVIEW

File: app/src/main/kotlin/org/virtuslab/bazelsteward/app/App.kt [hunk 1]
```diff
@@ -1,5 +1,8 @@
 package org.virtuslab.bazelsteward.app
 
+import bazel.src.BazelUpdater
```
Comment (lukaszwawrzyk) [inline app/src/main/kotlin/org/virtuslab/bazelsteward/app/App.kt:3]: it is pretty suspicious package

File: bazel/src/BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,9 @@
+library(
+    srcs = glob(["**/*.kt"]),
```
Comment (lukaszwawrzyk) [inline bazel/src/BUILD.bazel:2]: following maven layout we would have bazel/src/main/kotlin and the BUILD file would be located at bazel/src/main/

File: bazel/src/BazelVersionFileSearch.kt [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import kotlin.io.path.listDirectoryEntries
+import kotlin.io.path.name
+import kotlin.io.path.readText
+
+class BazelVersionFileSearch(config: Config) {
+
+  private val fileNames = setOf(".bazelversion", ".bazeliskrc")
+
+  val bazelVersionFiles: List<BazelFileSearch.BazelFile> by lazy {
+    config.path.listDirectoryEntries().filter { fileNames.contains(it.name) }.map { BazelFileSearch.BazelFile(it, it.readText()) }
```
Comment (lukaszwawrzyk) [inline bazel/src/BazelVersionFileSearch.kt:14]: what is in config.path?

File: bazel/test/BUILD.bazel [hunk 4]
```diff
@@ -0,0 +1,7 @@
+unit_tests(
+    srcs = glob(["**/*.kt"]),
+    test_package = "org.virtuslab.bazelsteward.bazel",
```
Comment (lukaszwawrzyk) [inline bazel/test/BUILD.bazel:3]: how does it work, if. your tests are not in this pacakage?

File: app/src/main/kotlin/org/virtuslab/bazelsteward/app/App.kt (hunk 1 above)
Comment (kamildoleglo) [reply app/src/main/kotlin/org/virtuslab/bazelsteward/app/App.kt:3]: image

File: bazel/src/BazelVersionFileSearch.kt (hunk 3 above)
Comment (kamildoleglo) [reply bazel/src/BazelVersionFileSearch.kt:14]: It should point to the repository root

File: bazel/test/BUILD.bazel (hunk 4 above)
Comment (kamildoleglo) [reply bazel/test/BUILD.bazel:3]: No idea actually

File: bazel/src/BUILD.bazel (hunk 2 above)
Comment (kamildoleglo) [reply bazel/src/BUILD.bazel]: Done

File: bazel/src/BazelVersionFileSearch.kt (hunk 3 above)
Comment (lukaszwawrzyk) [reply bazel/src/BazelVersionFileSearch.kt]: id call it workspaceRoot then. Now it sounds like path to the config. But it is out of scope ofc.

File: bazel/test/BUILD.bazel (hunk 4 above)
Comment (lukaszwawrzyk) [reply bazel/test/BUILD.bazel]: I actually meant that I doubt if it works at all :p

**Review by lukaszwawrzyk — APPROVED** [review]

File: bazel/test/BUILD.bazel (hunk 4 above)
Comment (kamildoleglo) [reply bazel/test/BUILD.bazel]: Ah, maybe it didn't :P

**Pull request merged**